test/parallel/test-http2-util-headers-list.js

```diff
@@ -154,6 +154,17 @@ const {
   );
 }
 
+{
+  // Arrays containing a single set-cookie value are handled correctly
+  // (issue #16452)
```

Comment:
Nit: can you please use the full URL: `https://github.com/nodejs/node/issues/16452`?